tutorials/epochs/plot_10_epochs_overview.py

```diff
@@ -97,18 +97,14 @@
 #         events_from_file = mne.read_events(sample_data_events_file)
 #
 #
-# Creating an :class:`~mne.Epochs` object
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-#
 # The :class:`~mne.io.Raw` object and the events array are the bare minimum
```

Comment:
I'm not sure of the markup styling, but somehow this section is embedded inside a block of code as a very long comment.  I don't think that's what you want.